requirements.txt

```diff
@@ -23,15 +23,15 @@ cffi==1.14.4
     # via cryptography
 chardet==3.0.4
     # via requests
-click-plugins==1.1.1
-    # via fiona
 click==7.0
     # via
     #   click-plugins
     #   cligj
     #   fiona
     #   pip-tools
     #   presto-python-client
+click-plugins==1.1.1
+    # via fiona
```

Comment:
How was pip-tools sorting dependencies before?

Reply:
I've no idea! I did the re-compile in a separate commit though so it's easy to verify there are no actual version changes.